File: ManageCertificate/Entites/RefInventory.cs
using System;
using System.Collections.Generic;


namespace Entites;
public partial class RefInventory
{
    public int InventoryId { get; set; }

    public int? CouncilId { get; set; }

    public int? CertificateId { get; set; }

    public int? Year { get; set; }

    public int? Inventory { get; set; }

    public virtual RefCertificateType? Certificate { get; set; }

    public virtual RefCouncil? Council { get; set; }
}

File: ManageCertificate/ManageCertificate/wwwroot/client/src/Models/RefInventory.ts
import { RefCertificateType } from "./RefCertificateType";
import { RefCouncil } from "./RefCouncil";

export interface RefInventory {
    inventoryId: number;
    councilId?: number;
    certificateId?: number;
    year?: number;
    inventory?: number;
    certificate?: RefCertificateType;
    council?: RefCouncil;
}

